conf/messages.cy

```diff
@@ -299,7 +299,7 @@ pages.how-it-works.lede-pre1.part2=yn caniatáu i chi drosglwyddo £{0} o’ch L
 pages.how-it-works.lede-pre1.link-text=Lwfans Priodasol
 pages.how-it-works.lede-pre2=Gall hyn ostwng treth eich priod neu bartner sifil hyd at £{0} y flwyddyn dreth hon (6 Ebrill i 5 Ebrill y flwyddyn nesaf).
 pages.how-it-works.lede-pre4=Er mwyn elwa fel pâr, mae angen i’ch incwm fod yn llai nag incwm eich partner ac yn is na £{0}  y flwyddyn.
-pages.how-it-works.lede-pre5=Gallwch ôl-ddyddio’ch cais i gynnwys unrhyw flwyddyn dreth er 5 Ebrill {0} pan oeddech yn gymwys ar gyfer Lwfans Priodasol.
+pages.how-it-works.lede-pre5=Gallwch ôl-ddyddio’ch hawliad i gynnwys unrhyw flwyddyn dreth o {0} ymlaen pan oeddech yn gymwys i gael Lwfans Priodasol.
```

Comment:
I believe the placeholder in this scenario just captures the relevant year. 
Looking at the Welsh translation it should have "dreth o 6 Ebrill" before the placeholder.